lib/Quantization/Quantization.cpp

```diff
@@ -453,6 +450,32 @@ static Node *quantizeNode(Function *F, Node *node,
         F->createTopK(topK->getName(), quantizedInputs[0], topK->getK());
     break;
   }
+  case Kinded::Kind::TanhNodeKind: {
+    assert(quantizedInputs.size() == 1 && "Invalid number of inputs");
+
+    // Quantized tanh operator expects input to be in a certain floating point
+    // range. This operator works based on the precomputed table and has to
+    // process input in a range of [-3.0, 3.0]. The output quantization
```

Comment:
Maybe add something here about why `[-3.0, 3.0]` is chosen -- I am assuming this is because `tanh` asymptotically approaches `[-1.0, 1.0]` and is already at `+/-.995` at `+/-3.0`

Reply:
Yup, this and also the more narrow the range the less quantization error to represent that range.